app/controllers/citizens/legal_aid_applications_controller.rb

```diff
@@ -4,28 +4,53 @@ class LegalAidApplicationsController < BaseController
     # User passes in the Secure Id at the start of the journey. If login succeeds, they
     # are redirected to index where the first page is displayed.
     def show
-      sign_out current_provider if provider_signed_in?
-      secure_id = params[:id]
-
-      legal_aid_application = LegalAidApplication.find_by_secure_id!(secure_id)
-
-      session[:current_application_ref] = legal_aid_application.id
-
-      sign_applicant_in_via_devise(legal_aid_application.applicant)
-      redirect_to citizens_legal_aid_applications_path
+      if find_legal_aid_application.error == :expired
+        expired
+      elsif application.completed_at.present?
+        completed
```

Comment:
If an `if` statement get to the point where you need to use an `elsif`, that's a smell! There is usually a better way of doing it. Here I'd suggest:
```ruby
return expired if find_legal_aid_application.error == :expired
return completed if application.completed_at.present?

start_applicant_flow
```